### DISCUSSION TITLE
My application verifying token in a loop when I refresh my browser when browser inspect tool is opened.

### DISCUSSION THREAD

**Discussion by suvel** [keycloak/keycloak#17424] [Q&A]
I have a React application that is getting reloaded in a loop when I press the refresh button, I observed the string "&state.." is getting added and removed from the URL in a loop.  I am able to reproduce this issue through these steps.

1. Open the application in Chrome/Brave (**not reproduce able in Edge**)
2. Login through Keyclock
3.  verify the token is stored in the cookie
4. click on the refresh button on the browser

Dependencies 

- "react": "17.0.2"
- "keycloak-js": "18.0.1"
- "react-router-dom": "6.3.0" (using HashRouter)

Keyclock configuration

```
onLoad: "login-required",
```

Please do let me know if more information is needed, thanks in advance.

**Comment by jonkoops** [comment]
We'll need a more reproducible example if you want to get an answer, I don't think anyone will be able to answer your question without it. As always I recommend that you use the latest version of both Keycloak and Keycloak JS, as your issue might already have been resolved.

**Reply by suvel** [reply]
Sure, I will check with the latest version and provide a more reproducible example.

**Comment by darius-m** [comment]
If there is an issue when the inspect tool is used (assuming this doesn't happen otherwise), perhaps you are disabling caches and the requests are load balanced to different backend Keycloak servers? This is a common issue if the Infinispan cluster between Keycloak nodes fails to synchronize the authentication caches.

**Reply by suvel** [reply]
Thanks for answering. 
I'm not sure about this, have checked with my peers. But if this issue caused due to how the backend is managing the request, it should be also replicated in Edge right?
